### DISCUSSION TITLE
API doesn't return list team projects

### DISCUSSION THREAD

**Discussion by alekspolskiy** [community/community#24306] [Apps, API and Webhooks]
<p>List team projects (<a href="https://docs.github.com/en/rest/reference/teams#list-team-projects" class="inline-onebox" rel="noopener nofollow ugc">Teams - GitHub Docs</a>) doesn’t work.<br>
Return requests.exceptions.HTTPError: 404 Client Error: Not Found for url:<br>
Url is correct</p>

**Comment by byrneh** [comment, marked as answer]
<p>Hi, <a class="mention" href="/alekspolskiy">@alekspolskiy</a>, welcome to the community</p>
<p>If it exists more often than not the solution for API based access 404s will be either</p>
<ul>
<li>the user does not have the needed permissions</li>
<li>the personal access token permission scope is not correct</li>
</ul>
<p>GitHub returns a 404 not found, rather than reveal any information about something you do not have permission on, even the fact that it may exist (this is GitHub following security good practise, but can be confusing)</p>
<p>You can edit the permissions scope of your token in the GUI<br>
Let us know if this solves you problem</p>

**Comment by alekspolskiy** [comment]
<p>was a problem with scopes. Thx</p>
